docs/aws/nRFCloudLocationServices.rst

```diff
@@ -80,3 +80,26 @@ Use the following commands:
     npx cdk deploy --all
 
 This will update the StateMachine, which resolves P-GPS request from devices to use the nRF Cloud API as a resolver.
+
+Ground Fix Services
+*******************
+
+You can enable your devices to request the nearest wi-fi access points using `nRF Cloud's Ground Fix Service API <https://api.nrfcloud.com/v1#tag/Ground-Fix>`_ in your deployment.
+
+To use the API, complete the following steps:
+
+1. Configure your service key.
+#. Enable the API using the CLI.
+#. Redeploy the stack.
+
+Use the following commands:
+
+.. parsed-literal::
+    :class: highlight
+
+    cat *location of your Ground Fix Service Key file* | ./cli.sh configure thirdParty nrfcloud groundFixServiceKey
+    ./cli.sh configure context stack nrfCloudGroundFix 1
+    npx cdk deploy --all
+
+This will update the StateMachine, which resolves the nearest wi-fi access points request from devices to use the nRF Cloud API as a resolver.
```

Comment:
```suggestion
This will update the StateMachine, which resolves the request for the nearest Wi-Fi access points from devices to use the nRF Cloud API as a resolver.
```

Reply:
Not so sure, but would this make sense if it was something like this:

"... which resolves the request for the nearest Wi-Fi access points from devices using the nRF Cloud API."

Reply:
I am not native speaker and I would prefer to follow the previous one.

Reply:
I am not native speaker and I would prefer to follow the previous one.